tests/integration-test.sh

```diff
@@ -44,7 +44,7 @@ if (echo $prBody | grep -q "\[parallel jobs="); then
     jobs=$(echo $parallel | awk -F"\[parallel jobs=" '{sub(/\].*/,"",$2);print $2}')
 fi
 
-if [[ $DRONE_BRANCH == "master" || $DRONE_BRANCH == "releases/"* ]] && [[ $DRONE_REPO == "vmware/vic" ]] && [[ $DRONE_BUILD_EVENT == "push" ]]; then
+if [[ $DRONE_BRANCH == "master" || $DRONE_BRANCH == "releases/"* || $DRONE_BRANCH == "feature/robo" ]] && [[ $DRONE_REPO == "vmware/vic" ]] && [[ $DRONE_BUILD_EVENT == "push" ]]; then
```

Comment:
This change probably shouldn't be merged, right?

Reply:
Yep, this was meant only for the feature branch.

Reply:
@zjs this is addressed in https://github.com/vmware/vic/pull/7689